.github/workflows/regenerate_english_userDocs_translation_source.yml

```diff
@@ -7,15 +7,19 @@ on:
     paths:
       - 'user_docs/en/*.md'
 
+permissions:
+  contents: write
+
 jobs:
   rebuild-translation-source:
     runs-on: windows-2025
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
       with:
         fetch-depth: 0
+        persist-credentials: true  # sets origin with GITHUB_TOKEN
```

Comment:
using the default github token means that any pushes to beta will not trigger a CI build. I think we should be pushing using nvaccessAuto credentials so beta builds will go out from pushes

Reply:
Additionally, we can create a token which doesn't expire.